src/Calendar.jsx

```diff
@@ -344,10 +344,13 @@ export default class Calendar extends Component {
   }
 
   onMouseOver = (value) => {
-    this.setState({ hover: value });
+    const { hover } = this.state;
+    if ((hover && hover.getTime()) !== value.getTime()) {
+      this.setState({ hover: value });
+    }
```

Comment:
That's not how you would like to compare fast-changing state entries. I suggest:

```js
this.setState(prevState => {
  if (prevState.hover && (prevState.hover.getTime() === value.getTime())) {
    // Abort updating state
    return null;
  }

  return { hover: value };
});
```

Reply:
fixed!